GridKit/Model/PowerElectronics/SystemModelPowerElectronics.hpp

```diff
@@ -81,11 +81,13 @@ namespace GridKit
     PowerElectronicsModel()
     {
       // Set system model parameters as default
-      rel_tol_         = 1e-4;
-      abs_tol_         = 1e-4;
-      this->max_steps_ = 2000;
+      rel_tol_                 = 1e-4;
+      abs_tol_                 = 1e-4;
+      this->max_steps_         = 2000;
       // By default don't use the jacobian
```

Comment:
```suggestion
      max_steps_         = 2000;
// By default don't use the jacobian
```
No need to use `this->...` with the member variable convention we use consistently in the code.